src/Tribe/Timezones.php

```diff
@@ -94,6 +94,36 @@ public static function wp_timezone_abbr( $date ) {
 	}
 
 	/**
+	 * Generates a timezone string from a UTC offset
+	 *
+	 * @param string $offset UTC offset
+	 *
+	 * @return string
+	 */
+	public static function generate_timezone_string_from_utc_offset( $offset ) {
+		if ( ! self::is_utc_offset( $offset ) ) {
+			return $offset;
+		}
+
+		// ensure we have the minutes on the offset
+		if ( ! strpos( $offset, ':' ) ) {
+			$offset .= ':00';
+		}
+
+		$offset = str_replace( 'UTC', '', $offset );
+
+		list( $hours, $minutes ) = explode( ':', $offset );
+		$seconds = $hours * 60 * 60 + $minutes * 60;
+
+		$timezone = timezone_name_from_abbr( '', $seconds, 1 );
```

Comment:
Mostly just curious - but why prefer testing for timezones with DST in effect before looking for non-DST zones?